spec/acceptance/ace/transport_app_spec.rb

```diff
@@ -46,15 +44,14 @@ def app
   ##################
   describe '/execute_catalog' do
     describe 'success' do
-      let(:certname) { 'vvtzckq3vzx995w.delivery.puppetlabs.net' }
-
-      it 'returns 200 with empty body when success' do
+      it 'returns 200 with `report_generated` status' do
         post '/execute_catalog', JSON.generate(execute_catalog_body), 'CONTENT_TYPE' => 'text/json'
         expect(last_response.errors).to match(/\A\Z/)
         expect(last_response).to be_ok
         expect(last_response.status).to eq(200)
         result = JSON.parse(last_response.body)
-        expect(result).to eq({})
+        expect(result['certname']).to eq('localhost')
+        expect(result['status']).to eq('report_generated')
```

Comment:
how much effort would it be to also run the PQL query for reports to prove that the catalog was executed successfully and communication with puppetserver was successful?